msteams-platform/includes/teamjs-version-details.md

```diff
@@ -1,7 +1,7 @@
 |                  |[TeamsJS](/javascript/api/overview/msteams-client) version | [App manifest](../resources/schema/manifest-schema.md) version| Next steps|
 |------------------|---------|--------|---|
 |**Teams apps extended to Office/Outlook**| TeamsJS v.2.0 or later  | **1.13** or later | [Extend a Teams app to run across Microsoft 365](../m365-apps/extend-m365-teams-personal-tab.md) or [Create a new Microsoft 365 app](../m365-apps/extend-m365-teams-personal-tab.md#quickstart) |
-|**Existing Teams-only apps**| Update to TeamsJS v.2.0 when possible (v.1.12 is still supported*)  | 1.12 | [Understand TeamsJS backwards compatibility](../tabs/how-to/using-teams-client-sdk.md#backwards-compatibility) and [Update to TeamsJS v.2.0](../tabs/how-to/using-teams-client-sdk.md#updating-to-the-teams-client-sdk-v200)|
+|**Existing Teams-only apps**| Update to TeamsJS v.2.0 when possible (v.1.12 is still supported*)  | 1.12 | [Understand TeamsJS backwards compatibility](../tabs/how-to/using-teams-client-sdk.md#backwards-compatibility) and [Update to TeamsJS v.2.0](../tabs/how-to/using-teams-client-sdk.md#updating-to-the-latest-teams-client-version)|
 |**New Teams-only apps**| TeamsJS v.2.0 or later | 1.12 | [Create a new Teams app using Teams Toolkit](../toolkit/create-new-project.md)|
```

Comment:
```suggestion
|**Teams apps extended to Office/Outlook**| TeamsJS v.2.0 or later  | **1.13** or later | [Extend a Teams app to run across Microsoft 365](../m365-apps/extend-m365-teams-personal-tab.md) or [Create a new Microsoft 365 app](../m365-apps/extend-m365-teams-personal-tab.md#quickstart) |
|**Existing Teams-only apps**| Update to TeamsJS v.2.0 when possible (v.1.12 is still supported*).| 1.12 | [Understand TeamsJS backwards compatibility](../tabs/how-to/using-teams-client-sdk.md#backwards-compatibility) and [Update to TeamsJS v.2.0](../tabs/how-to/using-teams-client-sdk.md#updating-to-the-latest-teams-client-version).|
|**New Teams-only apps**| TeamsJS v.2.0 or later | 1.12 | [Create a new Teams app using Teams Toolkit](../toolkit/create-new-project.md).|
```

Reply:
@v-ypalikila I'm not able to apply your suggestions because the current source already reflects the changes (and also it won't allow me to because I don't have write permissions). Everything here is good to go from my side--would you mind helping to merge (and [re]applying those edits if they arn't reflected on your side?) Thanks!

Reply:
Updated in the changes in the PR.